.changeset/old-papayas-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@apollo/server": patch
+---
+
+Use extensions for all imports to accommodate TS nodenext users
```

Comment:
can you be explicit about whether this is module:nodenext, moduleResolution:nodenext, or both?